Source/Core/EllipseGeometry.js

```diff
@@ -1,50 +1,56 @@
 /*global define*/
 define([
-        './BoundingSphere',
-        './Cartesian2',
-        './Cartesian3',
-        './Cartographic',
-        './ComponentDatatype',
-        './defaultValue',
-        './defined',
-        './DeveloperError',
-        './EllipseGeometryLibrary',
-        './Ellipsoid',
-        './GeographicProjection',
-        './Geometry',
-        './GeometryAttribute',
-        './GeometryAttributes',
-        './GeometryInstance',
-        './GeometryPipeline',
-        './IndexDatatype',
-        './Math',
-        './Matrix3',
-        './PrimitiveType',
-        './Quaternion',
-        './VertexFormat'
-    ], function(
-        BoundingSphere,
-        Cartesian2,
-        Cartesian3,
-        Cartographic,
-        ComponentDatatype,
-        defaultValue,
-        defined,
-        DeveloperError,
-        EllipseGeometryLibrary,
-        Ellipsoid,
-        GeographicProjection,
-        Geometry,
-        GeometryAttribute,
-        GeometryAttributes,
-        GeometryInstance,
-        GeometryPipeline,
-        IndexDatatype,
-        CesiumMath,
-        Matrix3,
-        PrimitiveType,
-        Quaternion,
-        VertexFormat) {
+    './BoundingSphere',
+    './Cartesian2',
+    './Cartesian3',
+    './Cartographic',
+    './ComponentDatatype',
+    './defaultValue',
+    './defined',
+    './DeveloperError',
+    './EllipseGeometryLibrary',
+    './Ellipsoid',
+    './GeographicProjection',
+    './Geometry',
+    './GeometryAttribute',
+    './GeometryAttributes',
+    './GeometryInstance',
+    './GeometryPipeline',
+    './IndexDatatype',
+    './Math',
+    './Matrix3',
+    './Matrix4',
+    './PrimitiveType',
+    './Quaternion',
+    './Rectangle',
+    './Transforms',
+    './VertexFormat'
+], function(
+    BoundingSphere,
+    Cartesian2,
+    Cartesian3,
+    Cartographic,
+    ComponentDatatype,
+    defaultValue,
+    defined,
+    DeveloperError,
+    EllipseGeometryLibrary,
+    Ellipsoid,
+    GeographicProjection,
+    Geometry,
+    GeometryAttribute,
+    GeometryAttributes,
+    GeometryInstance,
+    GeometryPipeline,
+    IndexDatatype,
+    CesiumMath,
+    Matrix3,
+    Matrix4,
+    PrimitiveType,
+    Quaternion,
+    Rectangle,
+    Transforms,
+    VertexFormat) {
```

Comment:
After merging in master, run `npm run sortRequires` in order to fix the formatting for modules.  It will modify whatever needs to change and you can just commit them all.